packages/aws-client/tests/test-DynamoDbSearchQueue.js

```diff
@@ -99,3 +99,40 @@ test.serial('DynamoDbSearchQueue can handle paging', async (t) => {
 
   t.is(returnedItemsCount, 11);
 });
+
+test.serial('DynamoDbSearchQueue returns results in query mode', async (t) => {
+  const bucket = randomString();
+
+  await Promise.all(range(11).map(() =>
+    awsServices.dynamodb().putItem({
+      TableName: t.context.tableName,
+      Item: {
+        bucket: { S: bucket },
+        key: { S: randomString() },
+      },
+    }).promise()));
+
+  const queue = new DynamoDbSearchQueue(
+    {
+      TableName: t.context.tableName,
+      Limit: 2,
```

Comment:
The `Limit` parameter to a Dynamo scan controls how many items are returned by an individual scan operation. By default, Dynamo will keep scanning until it has 1 MB of data, then return it, by you can specify `Limit` so that each scan operation only evaluates that many records before returning. 

In essence, doing this forces our test to do "paging" of the Dynamo results, which ensures that it is working for the "query" mode. But there's no reason it shouldn't, since that part of the code isn't dependent on the "mode"

Reply:
Sorry, I was really asking why it wasn't, say, 1 or 10 for the above purpose.    It's inconsequential in this context, was just curious.